pkg/common/catalog/closers.go

```diff
@@ -1,22 +1,25 @@
 package catalog
 
 import (
+	"errors"
 	"io"
 	"time"
 
-	"github.com/zeebo/errs"
 	"google.golang.org/grpc"
 )
 
 type closerGroup []io.Closer
 
 func (cs closerGroup) Close() error {
 	// Close in reverse order.
-	var errs errs.Group
+	var errs error
 	for i := len(cs) - 1; i >= 0; i-- {
-		errs.Add(cs[i].Close())
+		if err := cs[i].Close(); err != nil {
+			errs = errors.Join(errs, err)
+		}
```

Comment:
`Join()` discards nil error values so you should be able to do:
```suggestion
		errs = errors.Join(errs, cs[i].Close())
```